File: packages/patapata_core/android/src/main/kotlin/dev/patapata/patapata_core/PatapataPlugin.kt
/*
 * Copyright (c) GREE, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package dev.patapata.patapata_core

interface PatapataPlugin {
  val patapataName: String
  fun patapataEnable() {}
  fun patapataDisable() {}
}

File: packages/patapata_core/ios/Classes/PatapataPlugin.swift
// Copyright (c) GREE, Inc.
//
// This source code is licensed under the MIT license found in the
// LICENSE file in the root directory of this source tree.


public protocol PatapataPlugin {
    var patapataName: String { get }
    func patapataEnable()
    func patapataDisable()
}

public extension PatapataPlugin {
    func patapataEnable() {}
    func patapataDisable() {}
}
